src/validatefs/validatefs.c

```diff
@@ -0,0 +1,342 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <getopt.h>
+
+#include "blkid-util.h"
+#include "blockdev-util.h"
+#include "build.h"
+#include "chase.h"
+#include "device-util.h"
+#include "fd-util.h"
+#include "initrd-util.h"
+#include "main-func.h"
+#include "mountpoint-util.h"
+#include "nulstr-util.h"
+#include "parse-argument.h"
+#include "path-util.h"
+#include "pretty-print.h"
+#include "string-util.h"
+#include "utf8.h"
+#include "xattr-util.h"
+
+static char *arg_target = NULL;
+static char *arg_root = NULL;
+
+STATIC_DESTRUCTOR_REGISTER(arg_target, freep);
+STATIC_DESTRUCTOR_REGISTER(arg_root, freep);
+
+static int help(void) {
+        int r;
+
+        _cleanup_free_ char *link = NULL;
+        r = terminal_urlify_man("systemd-validatefs@.service", "8", &link);
+        if (r < 0)
+                return log_oom();
+
+        printf("%1$s [OPTIONS...] /path/to/mountpoint\n"
+               "\n%5$sCheck file system validation constraints.%6$s\n"
+               "\n%3$sOptions:%4$s\n"
```

Comment:
This contradicts your recent comment?
https://github.com/systemd/systemd/pull/36850#discussion_r2020594171